test/testdata/dsl/t_struct_override.rb

```diff
@@ -0,0 +1,17 @@
+# typed: true
+
+class Override < T::Struct
+  extend T::Sig
+
+  prop :foo, String
+
+  sig {params(foo: Integer).void}
+  def initialize(foo:)
+    @foo = foo.to_s
```

Comment:
This seems like a dangerous test. What do you think about making the body just be `puts 'in overridden initialize'`?

Reply:
Sure, but what do you mean by dangerous?